content/cucumber/configuration.md

```diff
@@ -95,8 +155,8 @@ So, if you have several profiles with similar values, you might do this:
    default: <%= common %> features
    html_report: <%= common %> --format html --out=features_report.html features
    ```
-
-## Environment Variables
+      
+<b>Environment Variables</b>
```

Comment:
For the Ruby page, this should definitely be a header

Reply:
The environment variables are only used in context of the configuration file though. So it's not a general section about environment variables. Just another way to configure the configuration?